### PR TITLE
Resolves #1548, added missing _canShowFeedback property

### PR DESCRIPTION
**Pull request by brian-learningpool**
This was causing an inconsistency with other core question plugins in the authoring tool.

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "adapt-contrib-matching",
-  "version": "2.1.1",
+  "version": "2.1.2",
```
Comment (moloko) [inline bower.json]: #99 already handles this (and version number in README too) so no need for you to do in this PR

Comment (brian-learningpool) [inline bower.json]: Yeah, thanks, I just noticed that when I created the PR -- there are quite a few waiting to go in.  It should still merge OK though.

Comment (oliverfoster) [inline bower.json]: it's going to need editing one way or another. either in a merge conflict or in a pr.

Comment (moloko) [inline bower.json]: #99 is now merged and it doesn't seem to be causing problems here so I think it's all good

Comment (oliverfoster) [inline bower.json]: nice

**Pull request merged**